README.md

```diff
@@ -48,4 +48,13 @@ A typical Prism response includes the following top level fields:
  - `stale` - whether prism believes that the data is stale based on the configured shelf life of sources
  - `staleSources` - a list of any sources that haven't been crawled successfully but may have contributed to the API response
  - `data` - the API response
- - `sources` - the list of sources that were interrogated to assemble the response
\ No newline at end of file
+ - `sources` - the list of sources that were interrogated to assemble the response
+
+ 
+ Adding an AWS account to Prism
+ ------------------------------
+
+* Create a new stack with `cloudformation/prism-role.template`. In the AWS account requiring Prism. Give it a Stack Name of PrismAccess. When prompted add the Prism account parameter.  The format is `arn:aws:iam::<account-number>:root`, please ask a Guardian developer for the account-number.
+* The output of the stack created is a Key named Role. Take the value of Role (`arn:aws:iam:[account-number]/PrismAccess-PrismRole-[code]`) and add it the spreadsheet listing all AWS Role values (ask a fellow Guardian developer for access to the sheet).
+* In DeployTools AWS account go the `config-deploy` DynamoDB table. Select the App `prism`, Stage `PROD` item. Choose `Text` view. Details of the AWS account requiring prism will need to be added here. There are many examples to follow, the format of the name must be the `[aws-account-name].role` with the output Role value of the step above. Save and close.
```

Comment:
Oh sorry I didn't get the initial notification (11 days ago). Yep I'll do that today.